_utilities/layout-grid.md

```diff
@@ -508,112 +414,249 @@ Add `grid-gap` to a grid row to add a gap (or gutter) between each column in the
 
 ### Variables
-Variables and maps determine the number of columns, gutter width, and media-query point at which to begin floating columns. We use variables to generate the predefined grid classes documented on this page, as well as for the custom mixins noted under [Spacing settings](#spacing-settings) and [Utilities settings](#utilities-settings).
+Variables and maps determine the number of columns, gutter width, and media-query point at which to begin floating columns. We use variables to generate the predefined grid classes documented on this page, as well as for the custom mixins noted under [spacing settings](#spacing-settings) and [utility mixins](#utility-mixins).
 
 #### Spacing settings
 
-{% highlight scss %}
-// Values are set as units tokens.
-
-$theme-column-gap-sm:               2px;
-$theme-column-gap-md:               2;
-$theme-column-gap-lg:               3;
-$theme-column-gap-mobile:           2;
-$theme-column-gap-desktop:          4;
-$theme-grid-container-max-width:    'desktop';
-$theme-site-margins-breakpoint:     'desktop';
-$theme-site-margins-width:          4;
-$theme-site-margins-mobile-width:   2;
-{% endhighlight %}
-
-#### Utilities settings
-
-{% highlight scss %}
-// Turn on or off breakpoints
-$theme-utility-breakpoints: (
-  'card':              false,   // 160px
-  'card-lg':           false,   // 240px
-  'mobile':            false,   // 320px
-  'mobile-lg':         true,    // 480px
-  'tablet':            true,    // 640px
-  'tablet-lg':         false,   // 880px
-  'desktop':           true,    // 1024px
-  'desktop-lg':        false,   // 1200px
-  'widescreen':        false,   // 1400px
-);
-{% endhighlight %}
-
-### Mixins
-Mixins can be used in conjunction with grid variables to add grid functionality to semantic component Sass.
-
-{% highlight scss %}
-// Creates a wrapper for a series of rows
-// $container-size can be mobile, mobile-lg, tablet, tablet-lg, desktop, desktop-lg, or widescreen
-@include grid-container;
-@include grid-container($container-size);
 
-// Creates a wrapper for a series of columns
-@include grid-row;
+{% include settings-table.html
+  settings=page.spacing_settings
+%}
 
-// Specify the width between columns
-// $gap-size can be sm, md, lg, 2px, 05, 1, 3, 4, 6
-@include grid-gap;
-@include grid-gap($gap-size);
-
-// Make the element full-width
-@include u-width(full);
-
-// Specify the number of columns the element should span
-// $columns can be auto, fill, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12
-@include grid-col;
-@include grid-col($columns);
-
-// Get fancy by offsetting or changing the display order
-// $offset can be none, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11
-@include grid-offset($offset);
-
-// $order can be first, last, initial, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11
-@include u-order($order);
-{% endhighlight %}
+<section id="utility-mixins" class="padding-top-4">
+  <h2 class="site-h2 margin-y-0">Utility mixins</h2>
+  {% include utilities/utility-mixin-intro.html %}
```

Comment:
Now that we are adding the standard mixin intro, can we remove the "Sass mixins" section?

Reply:
Was this resolved?

Reply:
Resolved in [57ef05b](https://github.com/uswds/uswds-site/pull/2276/commits/57ef05baeb9957e4c129ce108f06e3daf3357d36)